Arrays/3443MaximumManhattanDistanceAfterKChanges.py

```
@@ -0,0 +1,106 @@
+              current_sum=abs(result[-1][0])+abs(result[-1][1])
+              if current_sum>max_distance_so_far and (s[i]=='W' or s[i]=='E'):
+                  max_distance_so_far=current_sum 
+                  max_direction=s[i]
+                  max_index=i
+              if current_sum<minimum and (s[i]=='W' or s[i]=='E'):
+                  minimum=current_sum 
+                  min_direction=s[i] 
+                  min_index=i
+          print("Result NS: ",result)
+          return ((max_distance_so_far,max_direction,max_index),(minimum,min_direction,min_index))     
+      def maxDistance(self, s: str, k: int) -> int:
```

Flake8 (7.2.0)

[error] expected 1 blank line, found 0

(E301)